generator/src/test/java/org/apache/kafka/message/EntityTypeTest.java

```diff
@@ -17,15 +17,13 @@
 
 package org.apache.kafka.message;
 
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.Timeout;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
same as @chia7712  said .
if you can use assertThrows to replace  expectException . awesome

Reply:
I tried to use assertThrows to replace expectException.